pkg/kubelet/kubelet_pods.go

```diff
@@ -1001,6 +1002,60 @@ func (kl *Kubelet) removeOrphanedPodStatuses(pods []*v1.Pod, mirrorPods []*v1.Po
 	kl.statusManager.RemoveOrphanedStatuses(podUIDs)
 }
 
+func (kl *Kubelet) deleteOrphanedMirrorPods() {
```

Comment:
this func is copied or new?

Reply:
This func is new. It is located here to avoid import cycle:
```
can't load package: import cycle not allowed
package k8s.io/kubernetes/pkg/kubelet/pod
	imports k8s.io/kubernetes/pkg/kubelet/kuberuntime
	imports k8s.io/kubernetes/pkg/kubelet/cm
	imports k8s.io/kubernetes/pkg/kubelet/cm/cpumanager
	imports k8s.io/kubernetes/pkg/kubelet/status
	imports k8s.io/kubernetes/pkg/kubelet/pod
```